EPR.Antivirus/EPR.Antivirus.Application.UnitTests/Clients/SubmissionStatusApiClientTests.cs

```diff
@@ -195,7 +201,21 @@ private static bool IsValidReport(HttpContent? content, FileType fileType, ScanR
             return false;
         }
 
-        return report.AntivirusScanResult == scanResult
-               && report.BlobContainerName == (fileType == FileType.Pom ? PomContainerName : RegistrationContainerName);
+        var expectedBlobContainer = RegistrationContainerName;
+
+        if (fileType == FileType.Pom)
```

Comment:
You could use a switch expression here instead of multiple if statements

Reply:
Have modified to do as suggested